pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
     "aiozmq",
     "apischema==0.16.1",
     "immutables",
+    "pydantic",
```

Comment:
Does the version not matter for this?

Reply:
Unsure, was expecting @DiamondJoseph to correct it upon rebasing if it's wrong. I'm importing from v1 in all cases.

Reply:
I think we we're not using FastAPI can get away with using Pydantic v2. It's only where using fastapi<0.100 that we required pydantic<2